### DISCUSSION TITLE
Well Done"

### DISCUSSION THREAD

**Discussion by theemack** [givanz/Vvveb#7] [General]
Looking good - happy to see this has taken a bold step and new ventures await. FANTASTIC STUFF!

**Comment by givanz** [comment]
Thank you! :)

**Comment by diGiusepp3** [comment]
Love it!

**Reply by givanz** [reply]
Thank you!

**Comment by Twilight-Computer** [comment]
I have spent a lot of time looking at altnatives / WordPress / Joomla / Bootstrap and this by far is a gem. Found only by accident when someone else said in a post What about vvveb and I went looking - very pleased and happy with this approach. Keep up the good work. In the process of migrating my website to vvveb.

**Reply by givanz** [reply]
Thank you, let me know if you find any issues or have suggestions.

**Comment by Twilight-Computer** [comment]
A few comments from installing this multiple times in the last few days:

Installed on Rocky Linux 9 / Apache

Post installation I notice:

2 x Templates - Homepage and Home page. The first shows a full menu (Home 
/ About / Shop /Mega menu / Services / Blog / Contact), the second is 
missing Home / About / Shop  ?

Homepage Template has no currency displayed and when you click on the drop 
down arrow you get this error:
 
/var/www/html/testtwilight/storage/compile-templates/app_1_landing_index.html-edit 
on line 181

        Warning: Trying to access array offset on value of type null in 
/var/www/html/testtwilight/storage/compile-templates/app_1_landing_index.html-edid 
on line 181
        "type = "submit" value="USD" name="currency data-v-currency-codes> 
$ US Dollar

        Warning: Trying to access array offset on value of type null in 
/var/www/html/testtwilight/storage/compile-templates/app_1_landing_index.html-edid 
on line 181
        "type = "submit" value="EUR" name="currency data-v-currency-codes> 
  Euro

If you change the text on the Home page and save - you get a message 
saying it is saved, but when you view the page - no text changes (even 
after flushing the cache).

If you replace the logo on the Homepage (with a new logo of the same 
dimensions), the menu becomes invisible


Please advise if I can send any logging or other information to assist in 
working out these issues. I really want to use vvveb as my website.

The above examples can be viewed live at 
https://test.twilightcomputer.com.au


Regards, 

Gary Young | J.P. QBE  |  Twilight Computer Services




From:   "Givan" ***@***.***>
To:     "givanz/Vvveb" ***@***.***>
Cc:     "Twilight Computer Services" ***@***.***>, 
"Comment" ***@***.***>
Date:   19/03/2024 12:15 AM
Subject:        Re: [givanz/Vvveb] Well Done" (Discussion #7)



Thank you, let me know if you find any issues or have suggestions.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: 
***@***.***>

**Reply by givanz** [reply]
> 2 x Templates - Homepage and Home page.

This is the same issue as https://github.com/givanz/Vvveb/issues/98 ?

> Homepage Template has no currency displayed and when you click on the drop down arrow you get this error:

Thanks for the bug report, it's fixed in the latest update, you can redownload the zip and update `/app` folder and clear cache.

> If you change the text on the Home page and save - you get a message saying it is saved, but when you view the page - no text changes (even after flushing the cache).

This happens only on Rocky Linux 9 / Apache or on all installations?

Can you please try to manually empty `/public/page-cache` folder? One possible cause can be missing permissions and removing cache files fails silently.

You can also try to temporarily set maximum pemissions like 777 for `/public/page-cache`  and `/public/themes/landing` to see if it's file permission related.

There should be a warning displayed in `admin > tools > cache > full page cache` if write permissions is missing.


> If you replace the logo on the Homepage (with a new logo of the same dimensions), the menu becomes invisible

The logo is changed from page builder or from site settings?

> Please advise if I can send any logging or other information to assist in working out these issues. I really want to use vvveb as my website.

If it's possible please provide a link where I can check the invisible menu https://test.twilightcomputer.com.au is not loading
